templates/releases/releases.html

```diff
@@ -3,6 +3,10 @@
 
 {%- block title -%}{{ title | default(value="Releases - Docs.rs")}}{%- endblock title -%}
 
+{%- block meta -%}
+    <link rel="canonical" href="https://docs.rs/releases/{{ owner }}" />
```

Comment:
This template is used on _all_ release-pages, so will fail for the latest-releases or search-result page etc. 

I assume you wanted to canonicalize `/@owner` and `/owner` pages?